src/backend/distributed/sql/udfs/create_time_partitions/latest.sql

```diff
@@ -0,0 +1,50 @@
+CREATE OR REPLACE FUNCTION pg_catalog.create_time_partitions(
+    table_name regclass,
+    partition_interval INTERVAL,
+    end_at timestamptz,
+    start_from timestamptz DEFAULT now())
```

Comment:
It would be useful to add an error if start_from <= end_at

Otherwise this silently does nothing, and users might get confused.
```
select create_time_partitions('events', interval '1 month', '2021-01-01', '2022-01-01');
```